<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>690</width>
                <height>145</height>
            </rect>
        </property>
        <property name="maximumSize">
            <size>
                <width>690</width>
                <height>145</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="widget" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>230</width>
                            <height>145</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>230</width>
                            <height>145</height>
                        </size>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>下载源：</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QComboBox" name="cbox_source">
                                <item>
                                    <property name="text">
                                        <string>Xiaoyu源(MSL2)</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>官方源(下载慢 但是最新的)</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>SakuraMirror(旧版本)</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pbtn_ok1">
                                <property name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <property name="text">
                                    <string>确认</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="widget_2" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>230</width>
                            <height>145</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>230</width>
                            <height>145</height>
                        </size>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_2">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>要下载的端：</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QComboBox" name="cbox_type">
                                <property name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <item>
                                    <property name="text">
                                        <string>Paper</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Forge</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Vanilla</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Bukkit</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Spigot</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Mojang</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Nukkit</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>CubeRite</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>GlowStone</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Akarin</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>BunggeeCord</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>BukkitPlugins</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>TacoSpigot</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Cauldron</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Thermos</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>WaterFall</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QPushButton" name="pbtn_ok2">
                                <property name="enabled">
                                    <bool>true</bool>
                                </property>
                                <property name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <property name="text">
                                    <string>确认</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="widget_3" native="true">
                    <property name="minimumSize">
                        <size>
                            <width>230</width>
                            <height>145</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>230</width>
                            <height>145</height>
                        </size>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>要下载端版本：</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QComboBox" name="cbox_ver">
                                <property name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <item>
                                    <property name="text">
                                        <string>1.18.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.18</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.17.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.17</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.16.5</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.16.4</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.16.3</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.16.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.16.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.16</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.15.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.15.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.15</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.14.4</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.14.3</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.14.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.14.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.14</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.13.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.13.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.13</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.12.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.12.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.12</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.11.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.11.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.11</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.10.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.10.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.10</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.9.4</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.9.3</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.9.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.9.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.9</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.9</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.8</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.7</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.6</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.5</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.4</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.3</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.8</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.9</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.8</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.7</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.6</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.5</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.4</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.3</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.2</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7.1</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>1.7</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="pbtn_start_download">
                                <property name="cursor">
                                    <cursorShape>PointingHandCursor</cursorShape>
                                </property>
                                <property name="text">
                                    <string>开始下载</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
